_posts/0000-01-01-intro.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Bienvenido a nuestro juego de diapositivas!"
 ---
 
```

Comment:
This is a **great place** to introduce an *improvement*. "+1"
Visit **DanielDuende.es** for *more information*!](https://DanielDuende.es)

Reply:
Let's go ahead with this.